### DISCUSSION TITLE
Is there a way to turn off the up/down vote function for topics?

### DISCUSSION THREAD

**Discussion by allo-** [community/community#3270] [Discussions]
I would like to turn off the up/down vote function of discussion topics in own projects.

This is not reddit and for projects where it's useful I create a subreddit. Here I would like to enable discussions without people voting other people's topics up and down.

**Comment by jhpratt** [comment]
Not sure if you mean to disable it on a topic-by-topic basis or on a repo-wide basis, but I'd like to see the latter myself. Imo the :+1: and :-1: should suffice for anything this is intended to cover.

**Reply by allo-** [reply]
Yes. I'd like to disable all vote buttons and just have a normal forum.

Vote buttons create unwanted incentives and discourage people from asking questions that others may downvote. Using the emoji button should be enough to give feedback where it is needed.

**Comment by PranamBhat** [comment]
Hello!

Good one.

Actually, **GitHub** kept this feature to sort your discussions based on "**votes**" also it will be helpful when you/few user's want to sort the discussions depending on the "**vote counts**".

If the question/discussion is relevant, then user's will definitely "upvote". So there's no need to worry. 

If the question/discussion is not relevant, then user's will definitely "down-vote".

On either way, **GitHub** encourages proper usage of the platform.


Thank you!

**Reply by jhpratt** [reply]
"Proper use" depends on the repository. In the primary repository where I intend discussions to be used, there is no need for voting anywhere. _You don't know what I'm using discussions for. Don't tell me what "proper usage" is._

**Reply by allo-** [reply]
I know the idea of systems like Reddit, and I disagree that votes are always used correctly. There are places where the feature is useful, namely Reddit, but it is not at many other sites.
Either way, I do not want a discussion forum to use up and downvotes, but I want it to be just a forum.

You provide the feature as a beta to gather feedback.
My feedback is that I will stop using it for my projects when there is no option to disable the voting feature.

**Reply by olsonpm** [reply]
I got here looking for a way to disable topic voting.  I was thinking of using discussions as a way for users to ask questions about my project without raising an issue.  I don't want voting for this.  Emoji's would be fine though I don't really care either way about those.

A bummer we can't turn them off - I will find another solution

**Comment by Phanthom-Mekat** [comment]
GitHub Discussions does not support disabling upvote/downvote functionality.
Consider using GitHub Issues or setting Discussions to read-only for a voting-free format.
Encourage collaboration through comments to maintain engagement.
Focus on constructive feedback and discussions within your project's community.
Provide clear guidelines to ensure productive interactions.
